.github/labeler.yml

```diff
@@ -0,0 +1,10 @@
+component/zeebe:
+  - changed-files:
+    - all-globs-to-any-file: 
+      - '!operate/**'
+      - '!.github/workflows/operate-*'
```

Comment:
:question: Shared files will be labeled with both `component/zeebe` and `component/operate`, right? Might work well for a while but will become messy as we add more components. Maybe changes to shared files should rather not have component label?